components/constellation/pipeline.rs

```diff
@@ -530,13 +529,40 @@ impl UnprivilegedPipelineContent {
         Ok(child_process)
     }
 
+    // gaol is not supported on Windows yet, so this is just multiprocess
+    // without sandboxing
```

Comment:
My preference would be to add a cfg block to the existing non-Windows code rather than duplicating the non-sandbox pieces here.